index.js

```diff
@@ -76,4 +89,18 @@ process.on('SIGNINT', () => {
   });
 });
 
+// app.get("*", function (req, res) {
```

Comment:
this also looks like from a full stack application, I think this can be deleted too